lib/affirm/charge_event.rb

```diff
@@ -0,0 +1,20 @@
+module Affirm
+  class ChargeEvent
+    attr_reader :id, :transaction_id, :type, :created,
+      :amount, :fee, :order_id, :shipping_carrier, :shipping_confirmation,
+      :fee_refunded
+
+    def initialize(attrs)
+      @id                    = attrs["id"]
+      @transaction_id        = attrs["transaction_id"]
+      @type                  = attrs["type"]
+      @created               = attrs["created"]
+      @amount                = attrs["amount"]
+      @fee                   = attrs["fee"]
```

Comment:
Fee is an integer (it's expressed as cents). The reason I leaned this way was to allow me to easily update the `events